src/_help/doc-beta.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Beta
+---
+
+- TOC
+{:toc}
+
+Use the `x-beta` property inside an Operation, a schema or a parameter object to identify it as beta.
+The `x-beta` property is a boolean.
+
+Adding or removing this property is a strucutal change.
+A change on a beta component, can't be breaking.
+
+## Example usage
+
+Here under is an example of a beta Operation, a beta request body and a beta Schema attribute.
+
+```yaml
+  "paths": {
+    "/diffs": {
+      "post": {
+        "x-beta": true,
+        "description": "Create a diff between any two given API definitions",
+        "requestBody": {
+          "description": "The diff creation request object",
+          "content": {
+            "application/json": {
+              "schema": {
+                "x-beta": true,
+                "type": "object",
+                "properties": {
+                  "url": {
+                    "x-beta": true,
+                    "type": "string",
+                    "format": "uri",
+                    "description": "**Required** if `definition` is not present.\nCurrent definition URL. It should be accessible through HTTP by Bump.sh servers.\n"
+                  },
+                  ...
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+```
+
+The documentation displays a "Beta" flag on the components:
```

Comment:
You don't like the british curly quotes? They are the common topographic characters for English quotes :)
```suggestion
The documentation displays a “Beta” flag on the components:
```